src/lib/gql.ts

```diff
@@ -0,0 +1,6 @@
+import { optimism, optimismSepolia } from "viem/chains";
+
+export const gqlApiEndpoints = {
+  [optimism.id]: "https://optimism.easscan.org/graphql",
+  [optimismSepolia.id]: "https://optimism-sepolia-bedrock.easscan.org/graphql",
+} as const;
```

Comment:
👍

Reply:
is this being used anywhere though? not yet right?

Reply:
yup, not yet